raiden/settings.py

```diff
@@ -44,7 +44,7 @@
 
 DEFAULT_PATHFINDING_MAX_PATHS = 3
 DEFAULT_PATHFINDING_MAX_FEE = 1000
-DEFAULT_PATHFINDING_IOU_TIMEOUT = 500
+DEFAULT_PATHFINDING_IOU_TIMEOUT = 5000000000
```

Comment:
I wouldn't want to set the default this high. And please add a comment that gives a small explanation for the number.

Reply:
done